docs/narrative.rst

```diff
@@ -311,7 +312,7 @@ warning:
     >>> zope.deferredimport.sample7.x # doctest: +NORMALIZE_WHITESPACE
     docs/narrative.rst:1: DeprecationWarning:
                 x is deprecated. Import from sample1 instead
-      Deferred Imports
+      ==================
```

Comment:
This document does weird tricks with `__file__` and dynamically creating modules, so it winds up printing the first part of *this* file when a warning is issued. There's now a header line there instead of jumping right in to the title text.